### PR TITLE
Dca 2209 chromedriver check

### PR DESCRIPTION
**Pull request by SergeyMoroz0703**

### CODE REVIEW

File: app/util/conf.py [hunk 1]
```diff
@@ -48,6 +48,7 @@ def __init__(self, config_yml):
         self.custom_dataset_query = self.get_property('custom_dataset_query') or ""
         self.verbose = self.settings['verbose']
         self.total_actions_per_hour = self.get_property('total_actions_per_hour')
+        self.chromedriver_version = self.obj['modules']['selenium']['chromedriver']['version']
```
Comment (ometelytsia) [inline app/util/conf.py:51]: Why `self.chromedriver_version` is not in BaseAppSettings ?

File: app/util/pre_run/environment_compliance_check.py [hunk 2]
```diff
@@ -60,6 +72,21 @@ def validate_application_config(processors, app_name_upper, app_settings, min_de
...
 
+def validate_chromedriver_version(app_name, app_settings):
+    options = webdriver.ChromeOptions()
+    options.add_argument("--headless")
+    driver = webdriver.Chrome(options=options)
+    current_chrome_version = version.parse(
+        driver.capabilities['browserVersion'])
+    current_chromedriver_version = version.parse(
+        app_settings.chromedriver_version)
+    if current_chromedriver_version != current_chrome_version:
```
Comment (ometelytsia) [inline app/util/pre_run/environment_compliance_check.py:83]: I suppose we need to compare only the major version, but not the full version.
E.g. My current latest Chrome is `122.0.6261.112`, but the latest chomedriver is `122.0.6261.111`

File: app/util/conf.py (hunk 1 above)
Comment (SergeyMoroz0703) [reply app/util/conf.py:51]: Its necessary to avoid failure on any crowd run.

Comment (ometelytsia) [reply app/util/conf.py:51]: Initialize with default None?

File: app/util/pre_run/environment_compliance_check.py (hunk 2 above)
Comment (SergeyMoroz0703) [reply app/util/pre_run/environment_compliance_check.py]: Fixed

**Review by ometelytsia — APPROVED** [review]

**Pull request merged**